src/views/add-limited-partner-person.njk

```diff
@@ -29,7 +30,8 @@
     <div class="govuk-grid-column-two-thirds">
 
       <form class="form" action={{ props.currentUrl }} method="post">
-        <input type="hidden" name="pageType" value={{ props.pageType }}>      
+        <input type="hidden" name="pageType" value={{ props.pageType }}>     
+        <input type="hidden" name="partnershipType" value={{ partnershipType }}>
```

Comment:
is the hidden field needed? normally we add these to include the data when we submit the form back to the controller. If you just want to hide elements on teh page then the {% set partnershipType should be sufficient.

Reply:
It is needed we require it to be in the request body when the form is submitted.